smallrye-reactive-messaging-aws-sns/src/test/java/io/smallrye/reactive/messaging/aws/sns/AwsSnsTest.java

```diff
@@ -0,0 +1,89 @@
+package io.smallrye.reactive.messaging.aws.sns;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.eclipse.microprofile.reactive.messaging.Message;
+import org.eclipse.microprofile.reactive.streams.operators.SubscriberBuilder;
+import org.jboss.weld.environment.se.Weld;
+import org.jboss.weld.environment.se.WeldContainer;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.reactivestreams.Subscriber;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.reactivex.Flowable;
+
+public class AwsSnsTest extends AwsSnsTestBase {
+
+    private static final Logger LOG = LoggerFactory.getLogger(AwsSnsTest.class);
+    private WeldContainer container;
+    String topic = "sns-test";
+    String appUrl = "http://4befe2f5.ngrok.io";
```

Comment:
This is annoying, we need to find another way to deal with that. We cannot have a unit test relies on dynamic IPs. Is there any local mock we could use?

Did you try https://hub.docker.com/r/s12v/sns/?

Reply:
I think it is a good idea to evaluate and try it.